### DISCUSSION TITLE
difference between bolt 1.1 and the previous version

### DISCUSSION THREAD

**Discussion by hannwoei** [bitcraze/discussions#463] [Q&A] [answered]
hi, may I know the differences between the crazyflie bolt 1.1 and its previous version (bolt)? Thank you!

**Comment by tobbeanton** [comment, marked as answer]
Are you looking for more information then what is available on the [Bolt 1.1 product page](https://www.bitcraze.io/products/crazyflie-bolt-1-1/)?

**Reply by hannwoei** [reply]
Actually, I would like to know whether there is any update (software and hardware) for bolt 1.1, compared to the previous version. Can I use the current firmware on the previous version? I did searched on the Bitcraze websites but I could not find the relevant information.

**Reply by krichardsson** [reply]
No problem, the bolt 1.0 and 1.1 use the same firmware

**Reply by krichardsson** [reply]
Updated documentation in https://github.com/bitcraze/crazyflie-firmware/pull/1172

**Reply by hannwoei** [reply]
Thank you, guys!
